examples/chain-of-density/chain_of_density.py

```diff
@@ -0,0 +1,223 @@
+from pydantic import BaseModel, Field, model_validator, FieldValidationInfo
+from typing import List, Optional
+import spacy
+import instructor
+import openai
+from json import JSONDecodeError
+from pydantic import ValidationError
+
+instructor.patch()
+
+
+class MissingEntity(BaseModel):
+    """
+    An entity is a real-world object that's assigned a name - for example, a person, country a product or a book title.
+
+    A missing entity is:
+    - relevant to the main story,
+    - specific yet concise (5 words or fewer),
+    - novel (not in the previous summary),
+    - faithful (present in the article),
+    - anywhere (can be located anywhere in the article).
+    """
+
+    entity_name: str = Field(
+        ...,
+        description="This is the associated name with the entity that exists in the text",
+    )
+    reason: str = Field(
+        ...,
+        description="This is a short sentence which describes why we should include this new entity in the rewritten abstract",
+    )
+
+
+class OmittedEntity(BaseModel):
+    """
+    An entity is a real-world object that's assigned a name - for example, a person, country a product or a book title.
+    """
+
+    entity_name: str = Field(
+        ...,
+        description="This is an entity which was present in the previous summary and not in the newly generated summary",
+    )
+
+
+class MaybeOmittedEntities(BaseModel):
```

Comment:
Do you really need a maybe there. The fact that it's a list means if it's missing it just won't be in the list.

Reply:
I think moving forward will revert back to the original model which used omitted_entities and included_entities with default factories of list. This will have the same impact and we can do the `.missing` validation that we talked about,